### PR TITLE
Colorize contextual keyword as in import & export

### PR DESCRIPTION
**Pull request by XeCycle**
No idea how to test this.  Required?

### CODE REVIEW

**Comment by XeCycle** [comment]
Amended, added `from`.

File: js2-mode.el [hunk 1]
```diff
@@ -8870,12 +8865,10 @@ invalid export statements."
...
         (dolist (export exports-list)
           (push export children)))
-      (when (js2-match-token js2-NAME)
-        (if (equal "from" (js2-current-token-string))
-            (progn
-              (js2-unget-token)
-              (setq from-clause (js2-parse-from-clause)))
-          (js2-unget-token))))
+      (when (js2-match-contextual-kwd "from")
+        (progn
```
Comment (dgutov) [inline js2-mode.el]: Don't need `progn` inside `when`.

**Comment by dgutov** [comment]
Good idea, thanks.

Testing this isn't hard: you would call `get-text-property` at a given position. Maybe find that position with `(search-backward "as ")`. But we haven't been writing tests like that so far, so no, it's not required.

**Comment by XeCycle** [comment]
Fixed that progn, still no tests.  Btw another commit is here https://github.com/XeCycle/js2-mode/commit/292d1e5a82279e6dc69b6c410b02c9b08bdd07be, how do you like this one?

**Comment by dgutov** [comment]
Thanks!

>  Btw another commit is here XeCycle@292d1e5, how do you like this one?

Looks okay. Does it fix #295?

**Comment by XeCycle** [comment]
> Does it fix #295?

No I don't think them related.  Will open a PR for that anyway.

**Pull request merged**